packages/client/appmap.yml

```diff
@@ -0,0 +1,12 @@
+ordering: chronological
+command: yarn exec jest --no-cache
+recorder: jest
+exclude:
+- \[anonymous\]
+packages:
+- regexp: (^|/)node_modules/
+  enabled: false
+- regexp: ^../
+  enabled: false
+- regexp: ^
+  enabled: true
```

Comment:
Note that `enabled: true` is the default if we want to make the config file shorter. I still prefer the explicit `enabled` value though.

Reply:
Yeah, agreed.